assemblyscript.md

```diff
@@ -6,7 +6,17 @@ template = "page_lang"
 [extra]
 author = "Fermyon Staff"
 last_modified = "2022-03-10T21:50:50Z"
+
 ---
+
+- [AssemblyScript in WebAssembly](#assemblyscript-in-webassembly)
```

Comment:
Does the language guide now appear _only_ in the developer docs site?  If it's still available separately, how does this affect the appearance of the standalone site?

(And if it's _not_ available separately... why bother keeping a separate repo?)